fullstack-cert/react-projects/live-search/index.jsx

```diff
@@ -0,0 +1,46 @@
+const { useState, useEffect } = React;
+
+function LiveSearch() {
+  const [query, setQuery] = useState('');
+  const [results, setResults] = useState([]);
+
+  useEffect(() => {
+    if (query.trim() === '') {
+      setResults([]);
+      return;
+    }
+
+    const timeoutId = setTimeout(async () => {
+      try {
+        const response = await fetch(`https://api.tvmaze.com/search/shows?q=${query}`);
+        const data = await response.json();
+        setResults(data.map(item => item.show.name));
+      } catch (error) {
+        console.error("Error fetching data:", error);
+      }
+    }, 700);
+
+    return () => clearTimeout(timeoutId);
+  }, [query]);
+
+  return (
+    <div id="search-container">
```

Comment:
We should include a `label` and change this `div` to a `form` element like Tom did here
https://github.com/freeCodeCamp/demo-projects/pull/634/files#diff-7c4918320d422ec61f86f0ef216e7b1d0bfe339275fa162bbaed36a840efe760